conf/rubocop_without_rspec.yml

```diff
@@ -23,6 +23,9 @@ Bundler/GemComment:
     - 'github'
     - 'gist'
 
+Gemspec/DevelopmentDependencies:
```

Comment:
Should we use `EnforcedStyle: gemspec` so we can enforce the pattern we use?

Reply:
@erikkessler1 I'm ok with that, unless we happen to have some gems that use Gemfile and then we're causing the opposite problem.

Reply:
Even then, it seems like forcing them to use our convention of using the gemspec is what we want, right?